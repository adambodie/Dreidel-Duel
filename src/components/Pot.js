import React from "react"

const Pot = ({pot}) => {
	return(
		<div>
			<h3 style={{fontFamily: 'Vollkorn', margin: 0}}>Coins in the Pot:</h3>
			<h2 style={{fontFamily: 'Vollkorn', fontSize: '2em'}}>{pot}</h2>
		</div>
	)
}

export default Pot;
