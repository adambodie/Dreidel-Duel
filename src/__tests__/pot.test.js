import React from "react"
import renderer from "react-test-renderer"
import Pot from "../components/Pot"
import {shallow} from "enzyme"

describe("Pot", () =>
	it("renders correctly", () => {
		const tree = renderer.create(<Pot />).toJSON()
		expect(tree).toMatchSnapshot()
	}),

	it('renders an `h2` and `h3` text', () => {
		const wrapper = shallow(<Pot />);
		expect(wrapper.find('h2').length).toBe(1);
		expect(wrapper.find('h3').length).toBe(1);
	}),   
    it('has two child elements', () => {
		const wrapper = shallow(<Pot />);
		expect(wrapper.find('div').children().length).toBe(2);
	})
)
