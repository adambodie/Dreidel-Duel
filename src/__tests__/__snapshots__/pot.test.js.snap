// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pot renders correctly 1`] = `
<div>
  <h3>
    Coins in the Pot:
  </h3>
  <h2 />
</div>
`;
